FlowCrypt/src/main/java/com/flowcrypt/email/api/email/model/OutgoingMessageInfo.kt

```diff
@@ -48,18 +48,10 @@ data class OutgoingMessageInfo constructor(
     return getPublicRecipients() + getProtectedRecipients()
   }
 
-  fun getPublicRecipients(): List<String> {
-    val publicRecipients = mutableListOf<String>()
-    toRecipients?.let { publicRecipients.addAll(it.map { address -> address.address }) }
-    ccRecipients?.let { publicRecipients.addAll(it.map { address -> address.address }) }
-    return publicRecipients
-  }
+  fun getPublicRecipients() =
+    ((toRecipients ?: emptyList()) + (ccRecipients ?: emptyList())).map { it.address }
```

Comment:
This code is simple but doesn't seem like easy to read because of multiple parentheses
Maybe need to introduce `toAddresses` and `ccAddresses` as I mentioned before?

Reply:
Agree. I got carried away :)